Products/Archetypes/skins/archetypes/widgets/js/textcount.js

```diff
@@ -1,12 +1,13 @@
 <!-- Original:  Ronnie T. Moore -->
 <!-- Dynamic 'fix' by: Nannette Thacker -->
 function textCounter(field, countfield, maxlimit) {
-  var fieldval = jQuery(field).attr('value');
+  var fieldval = jQuery(field).val();
+  var countfield = jQuery('input[name="' + countfield + '"]');
```

Comment:
@mauritsvanrees The "countfield" that is passed to the function is actually name attribute value of the countfield element. Therefore, I create a variable that holds the real element, to get it's values etc.

The naming of the variable is not good, I will change this.